### ISSUE TITLE
Unpacking artifact issue

### ISSUE THREAD

**Issue by DavidFeldhoff** [microsoft/navcontainerhelper#1054]
**Describe the issue**
Similar to #1047. I cannot get it running to download the artifact and extract that one. Or better said the extract works, but the `Rename-Item` after that fails 99%. It seems that the `us-tmp` or `platform-tmp` folder is still in use of the operating system (probably our antivirus system which scans all zips). I cloned the navcontainerhelper and put a `Start-Sleep -Seconds 15` after the `Expand-Archive` and in front of the `Rename-Item` in `Download-Artifact.ps1`. Then it works. But I think that's not a solution, is it (even if it would be fine for me)? ;) Don't know what's the best practice in such a case.. 

**Script used to download artifact and cause the issue**
```
Download-Artifacts -artifactUrl (Get-BCArtifactUrl -country "us") -includePlatform
```

**Full output of scripts**
see screenshot

**Screenshots**
"Der Zugriff auf den Pfad ... wurde verweigert" = "The access to the path ... was denied."
![image](https://user-images.githubusercontent.com/53570297/86260925-38d6c100-bbbe-11ea-8057-83be339d879f.png)

**Additional context**

**Comment by freddydk** [comment]
did you try Flush-ContainerHelperCache and retry?

**Comment by freddydk** [comment]
Did you get the issue resolved?

**Comment by DavidFeldhoff** [comment]
I tried the `Flush-ContainerhelperCache`, but it didn't solve the issue.

**Comment by freddydk** [comment]
I think you are right in your assumption that this is a timing issue.
I will probably try the rename and if it fails - will pause and retry a few times.

**Comment by freddydk** [comment]
What I will do is the following (for both platform and app artifacts)

            while (Test-Path "$platformArtifactPath-tmp") {
                try {
                    Rename-Item -Path "$platformArtifactPath-tmp" -NewName ([System.IO.Path]::GetFileName($platformArtifactPath)) -Force
                }
                catch {
                    Write-Host "Unable to get exclusive access to $platformArtifactPath-tmp - waiting 10 seconds and retrying..."
                    Start-Sleep -Seconds 10
                }
            }

**Comment by DavidFeldhoff** [comment]
Thanks :)

**Comment by freddydk** [comment]
fixed in 0.7.0.11
